operators/src/source/ogr_source/mod.rs

```diff
@@ -1389,38 +1389,38 @@ pub trait FeatureCollectionBuilderGeometryHandler<G>
 where
     G: Geometry,
 {
-    fn push_generic_geometry(&mut self, geometry: G) -> Result<()>;
+    fn push_generic_geometry(&mut self, geometry: G);
 }
 
 impl FeatureCollectionBuilderGeometryHandler<MultiPoint>
     for FeatureCollectionRowBuilder<MultiPoint>
 {
-    fn push_generic_geometry(&mut self, geometry: MultiPoint) -> Result<()> {
-        self.push_geometry(geometry).map_err(Into::into)
+    fn push_generic_geometry(&mut self, geometry: MultiPoint) {
+        self.push_geometry(geometry);
```

Comment:
Why is there no option to fail necessary?

Reply:
after the arrow update, the `append` method on the builder can no longer fail. This saves us a lot of error handling.